src/nidcpower/metadata/enums.py

```diff
@@ -566,4 +566,40 @@
             },
         ],
     },
+    'MeasurementTypes': {
+        'values': [
+            {
+                'name': 'MEASURE_CURRENT',
+                'value': 0,
+                'documentation': {
+                    'description': 'The device measures current.',
```

Comment:
Where does this documentation come from? Was this file generated locally using the internal tools?

Reply:
Documentations are taken from session.py . Updated internal files to generate with same documentation.